src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
The check for `undefined` is correct as per the task requirements. However, be aware that in other contexts, this might not be sufficient to handle all falsy values unless explicitly intended.